eko/run-nlo.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import logging
+import pathlib
+import sys
+
+import numpy as np
+import pandas as pd
+import yaml
+from banana import toy
+from cfg import (
+    ffns_operator,
+    ffns_theory,
+    vfns_labels,
+    vfns_operator,
+    vfns_rotate_to_LHA,
+    vfns_theory,
+    xgrid,
+)
+
+import eko
+from eko.runner.managed import solve
+from ekobox import apply
+from ekomark.benchmark.external.LHA_utils import here as there
+
+_sqrt2 = float(np.sqrt(2))
+
+
+# reference values
+with open(there / "LHA_polarized.yaml", encoding="utf-8") as o:
+    ref_data = yaml.safe_load(o)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("scheme", help="FFNS or VFNS?")
+    parser.add_argument("sv", help="scale variation: up, central, or down")
+    parser.add_argument("--rerun", help="Rerun eko", action="store_true")
+    parser.add_argument(
+        "-v", "--verbose", help="Print eko log to screen", action="store_true"
+    )
+    args = parser.parse_args()
+
+    # determine xif
+    if "central".startswith(args.sv):
+        xif = 1.0
+        sv = "central"
```

Comment:
maybe let's avoid other conventions and just use `xif`  also in the file name

Reply:
Mmm, not sure ... we would start having $\sqrt 2$ floating around ... and I'm not 100% sure that all code use the same definition of `xif` (also the inverse is just fine ...) ... moreover this is used just for the tar - for the grids I would continue using table and part (even for NNLO)

Reply:
okay, fine